README.md

```diff
@@ -17,6 +17,15 @@ Add this to your `.tmux.conf` and run `Ctrl-I` for TPM to install the plugin.
 set -g @plugin 'omerxx/tmux-sessionx'
 ```
 
+### Installing in nix
```

Comment:
Was there some reason you closed this? I was about to submit a PR adding something like this before finding this, and your wording is a lot better than what I was going to write.

One thought I had: the footprint could be reduced with the use of a spoiler:

```markdown
<details>
  <summary>
  Nix
  </summary>
  ...
</details>
```